src/requirements.rs

```diff
@@ -0,0 +1,156 @@
+use super::ParseError;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, Default)]
+/// A requrirement. See [\<requires\>, \<recommends\>, &
+/// \<supports\>](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-relations)
+pub struct Requirements {
```

Comment:
Should be a enum I guess

Reply:
Now that it has `display_length`, `id` and `controls` fields, I think it is ok as a struct.